### PR TITLE
Fix SDL omits one of the URLs during retry sequence

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Fix SDL omits one of the URLs while cyclically changing during retry sequence.
This is copy of [Feature/cycling urls refactoring](https://github.com/smartdevicelink/sdl_core/pull/1258)

### CODE REVIEW

**Comment by LitvinenkoIra** [comment]
@VVeremjova @okozlovlux @LevchenkoS

**Comment by LitvinenkoIra** [comment]
Rebuild required

File: src/components/policy/policy_external/src/policy_manager_impl.cc [hunk 1]
```diff
@@ -1190,6 +1194,46 @@ void PolicyManagerImpl::SetDecryptedCertificate(
...
+    // Index of current application is OK, but all of its URL are sent,
+    // move to the next application
+    url_idx = 0;
+    if (++app_idx >= urls.size()) {
+      app_idx = 0;
+    }
+  }
+  const AppIdURL next_app_url = std::make_pair(app_idx, url_idx);
+
+  return next_app_url;
```
Comment (LevchenkoS) [inline src/components/policy/policy_external/src/policy_manager_impl.cc]: @LitvinenkoIra, you can return pair without of `next_app_url` variable.
Please, check everywhere.

**Comment by LitvinenkoIra** [comment]
Rebuild required

**Pull request closed without merging**